inscriptions/ISic020429.xml

```diff
@@ -198,18 +204,21 @@
                 <p><!--No translation currently agreed--></p>
             </div>
             <div type="commentary" xml:lang="en">
-                <p></p>
+                <p>According to the editor, the inscription could have been written by the same hand as <ref target="http://sicily.classics.ox.ac.uk/inscription/ISic020421">ISic020421</ref>
```

Comment:
I think as good practice, it would be correct here to replace 'editor' with 'Agostianiani', since any separate element could become detached fromt he other data, and it becomes rapidly unclear who the 'editor' in question might be.